cabra/src/pages/components/TaskListItem.tsx

```diff
@@ -77,6 +78,11 @@ export default function TaskListItem({ task }: Props) {
           </Card>
         </div>
       </Column>
+      {task.isBlocked && (
+        <p tw="flex items-center text-coolGray-500 pt-1">
+          Task is blocked by other task.
+        </p>
+      )}
```

Comment:
I think this label is unnecessary and kinda not esthetic.
I propose other method of visualization blocked task. What about task checkbox in other, maybe light red color?
Also tooltip for this checkbox would be great.
We are going to implement tooltip and alternative design of a checkbox for indirectly done task anyway.